src/main/java/org/opentripplanner/api/parameter/QualifiedMode.java

```diff
@@ -1,66 +1,35 @@
 package org.opentripplanner.api.parameter;
 
 import com.google.common.collect.Sets;
-import org.opentripplanner.routing.core.RoutingRequest;
-import org.opentripplanner.routing.core.TraverseMode;
 
 import java.io.Serializable;
-import java.security.InvalidParameterException;
 import java.util.Set;
 
 public class QualifiedMode implements Serializable {
     private static final long serialVersionUID = 1L;
     
-    public final TraverseMode mode;
-    public final Set<Qualifier> qualifiers = Sets.newHashSet();
+    public final String mode;
+    public final Set<String> qualifiers = Sets.newHashSet();
```

Comment:
Why is the Enum removed? It is part of the api and serve as documentation. We want to go in the directon of a more "type-Safe" direction, not the oposite way.

Reply:
I have now created new enums that are API specific and added back the type safety.